client/app/lib/constants/providerSpecs.coffee

```diff
@@ -0,0 +1,20 @@
+module.exports =
```

Comment:
let's make this a method takes `provider` as a parameter and returns list of available types